studio/components/interfaces/Settings/Database/ConnectionPooling.tsx

```diff
@@ -68,10 +68,8 @@ const ConnectionPooling: FC<Props> = () => {
     )
   }
 
-  const { project } = poolingConfiguration
-
   // for older projects
-  if (!project.pgbouncer_enabled && project.pool_mode == null)
+  if (!poolingConfiguration.pgbouncer_enabled && poolingConfiguration.pool_mode == null)
```

Comment:
can we update pages/api/database/[ref]/pooling.ts to remove `pool_mode` and `pgbouncer_xxx` stuff? i cant recall if this endpoint is being used in self-hosted, but just to keep this consistent first